### PR TITLE
chore:  Remove license section from configuration files

### PR DESCRIPTION
**Pull request by Yaminyam**
https://github.com/lablup/backend.ai/blob/733bdf6881ca72ac6815befee6bcc90dfcbcd635/configs/agent/sample.toml#L377

Similar to the agent.toml setup, the manager.toml requires license access information.

**Checklist:** (if applicable)

- [x] Milestone metadata specifying the target backport version

### CODE REVIEW

File: configs/manager/sample.toml [hunk 1]
```diff
@@ -243,3 +243,7 @@ enhanced-aiomonitor-task-info = false
 
 # Periodically sync container stat from Redis to the kernels.last_stat column.
 periodic-sync-stats = false
+
+
+[license]
```
Comment (jopemachine) [inline configs/manager/sample.toml]: In the development environment, this setting will cause an error indicating that it cannot connect to the license server. 
Wouldn't it be better to either comment out this setting only in `install-dev.sh` or find an alternative approach?

Comment (Yaminyam) [reply configs/manager/sample.toml]: If the license-related setting exists, but the associated plugin doesn't exist, no error is thrown.
In practice, we've been using the license-related setting in agent for a while now.

Comment (jopemachine) [reply configs/manager/sample.toml]: You're correct, please don't mind.

File: configs/manager/sample.toml [hunk 2]
```diff
@@ -243,3 +243,7 @@ enhanced-aiomonitor-task-info = false
...
+addr = { host = "127.0.0.1", port = 6099 }
```
Comment (HyeockJinKim) [inline configs/manager/sample.toml]: I haven't found the config code for the license, could you point me to where it is declared?

**Review by HyeockJinKim — APPROVED** [review]
LGTM

**Pull request merged**